datadog_checks_dev/datadog_checks/dev/tooling/commands/release/make.py

```diff
@@ -18,10 +18,11 @@
 
 
 def validate_version(ctx, param, value):
-    if re.match("^\\d+\\.\\d+\\.\\d+(-(rc|pre|alpha|beta)\\.\\d+)?$", value):
-        return value
+    if value:
```

Comment:
If there's no `value`, should this return nothing?

Reply:
It should return None, which is the default return value if there's no return. I explicitly added the return to make it clear!